docs/api/saved-objects/find.asciidoc

```diff
@@ -57,6 +57,10 @@ experimental[] Retrieve a paginated set of {kib} saved objects by various condit
   It should look like that savedObjectType.attributes.title: "myTitle". However, If you used a direct attribute of a saved object like `updatedAt`,
   you will have to define your filter like that savedObjectType.updatedAt > 2018-12-22.
 
+`aggs`::
+  (Optional, string) The aggs will support aggregation string with the caveat that your field from the aggregation will have the attribute from your type saved object, 
+  it should look like this: savedObjectType.attributes.field. However, If you use a direct attribute of a saved object like updatedAt, you will have to define your filter like this: savedObjectType.updatedAt.
```

Comment:
```suggestion
  it should look like this: savedObjectType.attributes.field. However, If you use a direct attribute of a saved object like updated_at, you will have to define your filter like this: savedObjectType.updated_at.
```

I see we also used `updatedAt` for the KQL filter docs above, could you change those to `updated_at` too?